src/main/java/core/basesyntax/dto/CSvParseService.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.dto;
+
+import core.basesyntax.models.Fruit;
+import core.basesyntax.models.FruitRecord;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class CSvParseService implements ParseService {
+    private static final String RECORD_SEPARATOR = "\r\n";
+    private static final String DATA_SEPARATOR = ",";
+    private static final int APPEND_TO_REMOVE_TITLE = 2;
+
+    @Override
+    public List<FruitRecord> convertStringDataIntoFruitRecordList(String datInString) {
```

Comment:
```suggestion
    public List<FruitRecord> parseFromCsv(String data) {
```

Reply:
Same with other methods and classes. Let's name variables and classes laconically